Sming/Components/rboot/.patches/rboot.patch

```diff
@@ -1,11 +1,14 @@
 diff --git a/Makefile b/Makefile
-index 0b43474..7cb71f4 100644
+index 0b43474..5176f8f 100644
```

Comment:
Would you rearrange this patch please, put rboot-api.c back at the end so comparison is easier?